pkgs/applications/emulators/emulationstation/default.nix

```
@@ -1,34 +1,72 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, cmake, curl, boost, eigen
-, freeimage, freetype, libGLU, libGL, SDL2, alsa-lib, libarchive
-, fetchpatch }:
+{ lib
+, SDL2
+, alsa-lib
+, boost
+, cmake
+, curl
+, eigen
+, fetchFromGitHub
+, fetchpatch
+, freeimage
+, freetype
+, libGL
+, libGLU
+, libarchive
+, pkg-config
+, stdenv
+, vlc
+, rapidjson
+, pugixml
+}:
 
 stdenv.mkDerivation {
   pname = "emulationstation";
-  version = "2.0.1a";
+  version = "2.12.0rp-dev";
```

```suggestion
  version = "2.11.2";
```

We don't like  to add such decorations when they are not present.

---

How does "being present" manifest? I used what is printed in `--version`

---

Here are the releases the upstream released until now:

https://github.com/RetroPie/EmulationStation/tags

You should follow the same convention they are using in that page, because this is where `fetchFromGitHub` picks the tarballs.